cmd/check.go

```diff
@@ -13,3 +18,25 @@ var checkCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(checkCmd)
 }
+
+func checkJunitAndWrite(cfg runtime.Config, results runtime.Results) error {
+	if viper.GetBool("junit") {
+		cfg.ResponseFormat = "junitxml"
```

Comment:
Do we want to copy this config instead of changing it? Not sure it matters in our execution, but I think it might be better to preserve the original.

Might want to check my logic - on the other hand, if we're not using pointers then we might be in the clear here because we passed the cfg into a function.

Reply:
It is being passed by value, not reference, so we are good here. That's the intent of this. It's a secondary formatter only being utilized when requested in addition to the primary formatter (which we don't allow the user to change anyway).

Reply:
So after looking at it. There's really no need to pass that in. I'll just create one and use that.